# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xf86-video-sunffb/hotfix-no-exec-stack.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- xf86-video-sunffb-1.2.3/src/VISmoveImage.s.vanilla	2025-10-02 19:27:38.200901775 +0200
+++ xf86-video-sunffb-1.2.3/src/VISmoveImage.s	2025-10-02 19:31:45.989699599 +0200
@@ -2709,3 +2709,5 @@
 RREPEND(rrep13e,rrep13r)
 RREPEND(rrep14e,rrep14r)
 RREPEND(rrep15e,rrep15r)
+
+	.section		.note.GNU-stack,"",@progbits
--- xf86-video-sunffb-1.2.3/src/ffb_asm.s.vanilla	2025-10-02 19:32:06.894279019 +0200
+++ xf86-video-sunffb-1.2.3/src/ffb_asm.s	2025-10-02 19:32:17.509711761 +0200
@@ -352,3 +352,5 @@
 	 mov	%o2, %o0
 
 	FIFO_WAIT(%o1, 3, FFB_LINE2_WAIT, 9b)
+
+	.section		.note.GNU-stack,"",@progbits
